service/src/main/java/bio/terra/workspace/service/policy/TpsApiDispatch.java

```diff
@@ -113,7 +113,16 @@ public TpsPaoGetResult getOrCreatePao(
       return pao.get();
     }
     // Workspace doesn't have a PAO, so create an empty one for it.
-    createPao(objectId, null, component, objectType);
+    try {
```

Comment:
I tried to get a test around this where we have some other policy tests and wasn't able to get the mocking to cooperate. I can try some other things, but felt like this was a bit of a rare situation and wasn't worth the extra effort. Let me know if you disagree and I'll see what else we can do to get coverage here.